src/expr/src/relation/func.rs

```diff
@@ -416,6 +416,41 @@ where
         })
 }
 
+fn string_agg<'a, 'b, I>(datums: I, temp_storage: &'a RowArena) -> Datum<'a>
```

Comment:
Is the `'b` needed? It's not referenced.

Reply:
Oh derp, great catch, ty